README.md

```diff
@@ -1,2 +1,57 @@
-# cozystack-website
-Cozystack.io website
+# Cozystack 🌐
+
+**Cozystack** is an open-source, cloud-native platform designed to simplify the management and deployment of custom cloud environments using **Kubernetes**. It offers seamless integration, high performance, and cost-efficiency, ideal for developers and organizations looking to scale applications with ease.
+
+## 🚀 Features
+
+- **Kubernetes-based infrastructure** 🛠️: Effortlessly deploy, manage, and scale applications.
+- **Cost-Efficient** 💰: Maximize performance while optimizing resource utilization.
+- **RESTful API**: Programmatic control over deployments and monitoring.
+- **Open-Source**: Community-driven development with no vendor lock-in.
+- **Pre-configured Monitoring** 📊: Built-in tools for tracking app performance.
+- **Multi-Cloud Support** ☁️: Integrates with various cloud providers for flexibility.
+
+---
+
+## 📝 Installation
+
+1. **Requirements**: Kubernetes, Docker, and Helm.
+2. **Quick Install**:
+    ```bash
+    curl -fsSL https://cozystack.io/install.sh | bash
```

Comment:
It does not work like that, istallation is done using Talos Linux and appliying installation manifest to it.